src/include/daos_task.h

```diff
@@ -69,6 +69,7 @@ typedef enum {
 	DAOS_OPC_CONT_CLOSE,
 	DAOS_OPC_CONT_DESTROY,
 	DAOS_OPC_CONT_QUERY,
+	DAOS_OPC_CONT_AGGREGATE,
```

Comment:
For task API, you are missing an update to src/client/api/task_internal.h

Reply:
Good catch, I'll fix it.